_includes/_inventory/resources.md

```diff
@@ -1,31 +1,31 @@
-## GCP Resource Coverage
+## Google Cloud Platform Resource Coverage
```

Comment:
Since you changed this heading this broke the intra-page [link](https://travis-ci.org/#L384).

You can fix this by adding `about/index.md` to this PR and fixing the link [here](https://github.com/GoogleCloudPlatform/forseti-security/blob/gh-pages/about/index.md#policy-scanner-link-_docsquickstartsscannerindexmd-) to be `#google-cloud-platform-resource-coverage` instead.